### PR TITLE
Mixtral

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

File: src/optimum/nvidia/hub.py [hunk 1]
```diff
@@ -181,17 +229,15 @@ def convert_and_build(
...
         if hf_model is None:
             LOGGER.debug(
                 f"Loading weights from {local_path} into the model ({cls.HF_LIBRARY_TARGET_MODEL_CLASS.__name__})"
             )
 
             hf_model = cls.HF_LIBRARY_TARGET_MODEL_CLASS.from_pretrained(
                 local_path,
-                device_map="auto",
-                max_memory=max_memory,
+                torch_dtype="auto",
```
Comment (fxmarty) [inline src/optimum/nvidia/hub.py:239]: shouldn't we always load in fp16/bf16? do we ever want to load in fp32? or the fp32 -> lower precision is handled later?

File: src/optimum/nvidia/utils/tests/__init__.py [hunk 2]
```diff
@@ -13,4 +13,5 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from .utils import nightly, requires_gpu, slow
+from .assertions import *
```
Comment (fxmarty) [inline src/optimum/nvidia/utils/tests/__init__.py:16]: we should avoid star imports if possible

**Review by fxmarty — APPROVED** [review]
LGTM

File: src/optimum/nvidia/hub.py (hunk 1 above)
Comment (mfuntowicz) [reply src/optimum/nvidia/hub.py:239]: I would say we want to load in the dtype the checkpoint is saved. 

`"auto"` will read it from the config and default to something else if not found

**Pull request merged**